### ISSUE TITLE
Can't go into a server

### ISSUE THREAD

**Issue by lafkpages** [SebOuellette/LiveBot#168]
#### What branch/release are you using?

- Branch/Release:
Latest

#### Errors

![image](https://user-images.githubusercontent.com/62298468/154832065-1046b63f-bc18-4ec9-ad3c-4d2acf1d5942.png)

#### Steps required to reproduce the bug

- Steps:
Open LiveBot, login and click on a server icon.

#### Screenshot (In most cases required)

- Screenshot:
![image](https://user-images.githubusercontent.com/62298468/154832065-1046b63f-bc18-4ec9-ad3c-4d2acf1d5942.png)

#### Description

I can't go into any server.

**Comment by nearlySplat** [comment]
This was supposedly fixed in 6c7e02f0a82e5c2c2a36bc1b497164a53a2ba76f. Are you on that release?

**Comment by lafkpages** [comment]
I downloaded the latest from leseases

**Comment by nearlySplat** [comment]
Are you sure the user LiveBot is being executed by has access to the directory it's in?

**Comment by lafkpages** [comment]
I'm on macOS. I just downloaded it and moved it into the Applications folder.

**Comment by lafkpages** [comment]
I'm on macOS. I just downloaded it and moved it into the Applications folder.

**Comment by SebOuellette** [comment]
Yeah in macos the solution still doesn't work , and as I'm not a mac developer/don't own a mac to test on, I'm not sure of the cause. Use the master branch for now instead of a release. You can follow the instructions on the readme to see how to do that

**Comment by Studio-Hawaii** [comment]
Move it into your applications folder, that fix worked for me
